### PR TITLE
INTERNAL-FIX: vsphere-refactor-follow-up

### PR DESCRIPTION
**Pull request by dfitzmau**
INTERNAL FIX that addresses feedback from PR https://github.com/openshift/openshift-docs/pull/73685

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-10048

Link to docs preview:
* [installing-restricted-networks-installer-provisioned-vsphere.adoc](https://74511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-restricted-networks-installer-provisioned-vsphere)
* [Prereq+Next steps installing-vsphere-installer-provisioned-network-customizations](https://74511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations#prerequisites_installing-vsphere-installer-provisioned-network-customizations)
* [Prereq+Next steps installing-vsphere-installer-provisioned-customizations](https://74511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-customizations#prerequisites_installing-vsphere-installer-provisioned-customizations)
* [Prereq+Next steps installing-vsphere-installer-provisioned](https://74511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned)
* [Prereq+Next steps installing-restricted-networks-vsphere ](https://74511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/upi/installing-restricted-networks-vsphere#prerequisites_installing-restricted-networks-vsphere_installing-restricted-networks-vsphere)
* [Prereq+Next steps installing-vsphere-network-customizations](https://74511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/upi/installing-vsphere-network-customizations)
* [Prereq+Next stepsinstalling-vsphere](https://74511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/upi/installing-vsphere#prerequisites_installing-vsphere_installing-vsphere)

### CODE REVIEW

File: installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc [hunk 1]
```diff
@@ -11,6 +11,7 @@ VMware vSphere instance by using installer-provisioned infrastructure.
 
 include::snippets/vcenter-support.adoc[]
 
+[id="prerequisites_installing-vsphere-installer-provisioned_{context}"]
```
Comment (kcarmichael08) [inline installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc:14]: Is the `_{context}` needed for an assembly? Looking at the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), I'm not sure it's needed here but I could be missing the reason.

**Review by kcarmichael08 — COMMENTED** [review]
One question about why some of the assembly ids have the `_{context}` part...I have not seen this before but I might not understand what it's doing here. 

LGTM!

**Comment by kcarmichael08** [comment]
added a label for 4.16, plmk if that's not correct!

**Comment by dfitzmau** [comment]
> added a label for 4.16, plmk if that's not correct!

Thanks. Correct. Updated!

File: installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc (hunk 1 above)
Comment (dfitzmau) [reply installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc:14]: Yes, sorry. I followed Mike's style.

**Pull request merged**